docs/40_node-operation/10_getting-started/40_peering.md

```diff
@@ -1,4 +1,165 @@
 ---
 title: Peering
-sidebar_class_name: sidebarhidden
 ---
+
+# Peering
+
+In a previous section, you deployed a local single node EOS blockchain. Although it is a fully functional blockchain, it still runs on a single node. To start grasping the full benefits of the EOS blockchain technology, you need to add more nodes. The [Bios Boot Tutorial](https://github.com/AntelopeIO/leap/tree/release/4.0/tutorials/bios-boot-tutorial) showcases how to deploy a multi node blockchain consisting of 21 producing nodes. In this section, you will learn about "peering", the network feature that makes it all possible. This is what allows to grow incrementally from a single node to a truly decentralized, geographically distributed, multi node EOS blockchain.
```

Comment:
I thought a local single node would be covered in the Node Configuration guide, or right after. Since it doesn't involve peering it makes sense to cover it before this guide? @nsjames

Reply:
How about referring to a local single node tutorial in the Node Tutorials section? "In the Node Tutorials section, you will deploy..."

Reply:
Fixed

Reply:
Actually, deploying a node isn't part of this flow. These are understanding the __concepts__, not launching. There will be three distinct journeys which launch things later, and API, a Block Producer, and a History Node. 

This guide should **only** talk about the concepts of peering. 

It also shouldn't lead any devs to guides which launch full chains, as this is before the end of their journey and they have more concepts to learn.

Reply:
Removed reference to tutorials and rewrote intro.